src/its_preselector/controlbyweb_web_relay.py

```diff
@@ -83,6 +83,23 @@ def get_digital_input_value(self, input_num: int) -> bool:
             raise ConfigurationException(f"Digital Input {input_num} does not exist.")
         return bool(int(digital_input.text))
 
+    def get_analog_input_value(self, input_num: int) -> bool:
+        """
+        Read float value from an analog input of the WebRelay.
+
+        :param input_num: Configured index of the desired analog input.
+        :raises ConfigurationException: If the requested analog input cannot be read.
+        :return: The desired analog input value.
+        """
+        input_string = str(input_num)
+        response = self.get_state_xml()
+        input_tag = f"analogInput{input_string}"
```

Comment:
The f-string syntax is redundant with typecasting from 2 lines above. Recommend deleting line 94.